src/loaders/directwrite.rs

```diff
@@ -471,15 +508,91 @@ impl Font {
                 }
             };
 
-            DWriteGlyphRunAnalysis::create(&glyph_run,
+            Ok(DWriteGlyphRunAnalysis::create(&glyph_run,
                                            1.0,
                                            None,
                                            rendering_mode,
                                            DWRITE_MEASURING_MODE_NATURAL,
                                            0.0,
-                                           0.0)
+                                           0.0)?)
+        }
+    }
+
+    /// Get font fallback results for the given text and locale.
+    /// 
+    /// Note: on Windows 10, the result is a single font.
+    pub fn get_fallbacks(&self, text: &str, locale: &str) -> FallbackResult {
```

Comment:
I think it would be nice to specify some examples for the locale parameter, like `"en-US"` or `"zh-TW"`, so that people don't try to write things like `"English"` :)